engine/cdr.go

```diff
@@ -315,6 +315,9 @@ func (cdr *CDR) exportFieldValue(cfgCdrFld *config.FCTemplate, filterS *FilterS)
 		var cdrVal string
 		switch cfgCdrFld.Path {
 		case utils.MetaExp + utils.NestingSep + utils.COST:
+			if cfgCdrFld.RoundingDecimals == 0 {
```

Comment:
We need separate var here since otherwise we will overwrite the config object from JSON.